MachineLearning_Engine/Compute/LinearRegression.cs

```diff
@@ -57,5 +60,21 @@ public static Tensor Infer(LinearRegression model, Tensor x)
         }
 
         /*************************************/
+
+        [Description("Projects the given input using a linear regression model")]
+        [Input("model", "The linear regressor model used for inference")]
+        [Output("")]
+        public static Output<List<double>, double> Coefs(LinearRegression model)
+        {
+            PyObject coefs = BH.Engine.MachineLearning.Compute.Invoke("LinearRegression.coefs", model);
```

Comment:
I believe you can get the attributes of the model directly.
```c#
double coef = model.SkLearnModel.GetAttr("coef_").As<double>();
double intercept = model.SkLearnModel.GetAttr("intercept_").As<double>();
return new Output<double, double> { Item1 = coef, Item2 = intercept };
```

Reply:
This is clearer. 
However, having the same issue that need a conversion between the variable given by .GetAttr and c# type.
If we print the coef we will get "[[ 2.78734161e-01 -2.13865448e-17]]" for a 2 dimension regression, and "[1.]" for the intercept. 
Instead of handling the conversion like I did, we should have a conversion method.

Reply:
You can use `BH.Engine.Python.Convert.IFromPython` if you don't know the type that you will receive. Or, in alternative, you can return a `Tensor` out of it
`Tensor coeff = new Tensor(model.SkLearnModel.GetAttr("intercept_"));`

Reply:
Great. Tensor then to list will work!